.github/workflows/build_rust_hello_world_reusable.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Test RustHelloWorld
+on:
+    workflow_call:
+      inputs:
+        cedar_policy_ref:
+          required: false
+          type: string
+        cedar_examples_ref:
+          required: false
+          default: "main"
+          type: string
+
+jobs:
+  build_and_test_rust_hello_world:
+    name: rust-hello-world
+    runs-on: ubuntu-latest
+    needs: get-branch-name
+    strategy:
+      matrix:
+        toolchain:
+          - stable
+    steps:
+steps:
+      - name: Checkout Cedar Examples
+        uses: actions/checkout@v3
+        with:
+          repository: cedar-policy/cedar-examples
+          ref: ${{ inputs.cedar_examples_ref }}
+      - name: Checkout cedar-policy
```

Comment:
Can we delete the `cedar-policy` checkout (and later the `printf "\npath = \"../cedar/cedar-policy\"" >> Cargo.toml`) now that we're adding the `[patch.crates-io]`?